### ISSUE TITLE
Why is length of audio (1.4.2) and auto updating content (2.2.2) different?

### ISSUE THREAD

**Issue by Wildebrew** [w3c/wcag3#181]
According to [WCAG 1.4.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html) audio can start automatically and autoplay for 3 seconds whereas, according to [WCAG 2.2.2](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html) video or auto updating content can start automatically and last for 5 seconds.

The 3 vs. 5 second difference is hard to measure and ultimately there is no reason why these should be different.

Why would a screen reader user be less patient than someone looking at the page? These time limits should be the same. Us blind people are somewhat known for being less than patient from time to time, but I think the 2 second diffrence creates unnecessary complexity for testers.

In the bigger picture, I am not sure why auto playing audio is in the "perceivable" section whereas auto updating content is in 2.2.2, operable.

Loud audio that can't be stopped makes it impossible to operate the page just like animations that prevent a screen reader from reading the page, or a person with a cognitive impairment from perceiving the page.

Ultimately I would recommend dropping the auto playing audio success criterion altogether and adding the auto playing audio requirement to 2.2.2 (or the equivalent in WCAG AG), these are all factors that prevent users from being able to perceive or operate the webpage as a hole and should be assessed together.
